cmd/kops/rollingupdatecluster.go

```diff
@@ -44,6 +44,8 @@ type RollingUpdateOptions struct {
 	NodeInterval    time.Duration
 	BastionInterval time.Duration
 
+	Headroom int64
```

Comment:
Trying to match terminology with core Kubernetes makes sense to me, though the current logic wouldn't imply something like `maxSurge` so it could be misleading. `maxSurge` would imply we're always adding extra nodes before terminating, but here we're terminating first and then replacing (after headroom that is). I think eventually we do want the rolling update logic to match that of pod rolling update right? When we do we should consider renaming. For now should we stick with something like `initialMaxSurge`?